fastlane/Fastfile

```diff
@@ -50,19 +54,22 @@ lane :upload_release do
 end
 
 desc 'Download the development signing certificates to this machine'
-lane :set_up_signing do
+lane :set_up_signing do |readonly: true|
+  require_env_vars!(*ASC_API_KEY_ENV_VARS, *CODE_SIGNING_STORAGE_ENV_VARS)
+
   sync_code_signing(
     platform: 'macos',
     app_identifier: APPLE_BUNDLE_IDENTIFIER,
     team_id: APPLE_TEAM_ID,
+    api_key: app_store_connect_api_key,
```

Comment:
While at it, I updated to our standard of using a runtime-generated API key  for interacting with the ASC API